moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_robot_distance_field.h

```diff
@@ -102,37 +103,37 @@ class CollisionRobotDistanceField : public CollisionRobot
   void checkSelfCollision(const collision_detection::CollisionRequest& req, collision_detection::CollisionResult& res,
                           const moveit::core::RobotState& state1, const moveit::core::RobotState& state2) const override
   {
-    ROS_ERROR_NAMED("collision_distance_field", "Not implemented");
+    RCLCPP_ERROR(LOGGER_COLLISION_DISTANCE_FIELD, "Not implemented");
```

Comment:
This class should have it's own logger. Perhaps 
`static rclcpp::Logger LOGGER_COLLISION_ROBOT_DISTANCE_FIELD = LOGGER_COLLISION_DISTANCE_FIELD.get_child("collision_robot_distance_field")`

Reply:
Changed both, for collision_robot_distance_field and collision_world_distance_field

https://github.com/ros-planning/moveit2/pull/65/commits/86c82b3e923f895cf57c5f8517a960ecd7c28e33
https://github.com/ros-planning/moveit2/pull/65/commits/6fc66933c5a5b4620f8c4a5708c308976afddcef
https://github.com/ros-planning/moveit2/pull/65/commits/2449e1e223c9b53379e690e547481d643fe5d929